os/include/tinyara/fs/fs_tmpfs.h

```diff
@@ -0,0 +1,228 @@
+/****************************************************************************
```

Comment:
I think we don't have to put header file in include folder.

Reply:
okay